src/sass/main.scss

```diff
@@ -94,3 +94,8 @@
 
 @import 'overrides/overrides.scss';
 @import 'overrides/print.scss';
+
+// Required because Google Auto-complete needs to appear above modal: https://github.com/twbs/bootstrap/issues/4160
```

Comment:
@Anoninnyc Please do not put random styles in this sheet – it's meant to be a directory for partials to keep things ordered and grouped properly. Name your styles 'appropriately' (ping me if you'd like guidance on this) and place them in a relevant scss partial.